src/viur/core/skeleton.py

```diff
@@ -1968,34 +1968,60 @@ def canCall(self) -> bool:
         return user is not None and "root" in user["access"]
 
     def dataSkel(self):
-        modules = ["*"] + listKnownSkeletons()
-        modules.sort()
         skel = BaseSkeleton().clone()
-        skel.module = SelectBone(descr="Module", values={x: translate(x) for x in modules}, required=True)
+
+        skel.skelname = SelectBone(
+            descr="Kind",
+            values={"*": "*"} | {name: translate(name) for name in sorted(listKnownSkeletons())},
+        )
+
+        skel.specific_key = StringBone(
```

Comment:
How about making this multiple?

And if you are already on this point, I would like to have module multiple too. I often start this for several but not all modules.

Reply:
Sure, this can be done in a separate PR by someone who invests a day or so into this.

Reply:
Why should we touch it twice when you're already working on the code?

It's just unnecessary extra work for everyone to create, test and review two PRs.